weed/storage/store.go

```diff
@@ -361,6 +361,12 @@ func (s *Store) SetStopping() {
 	}
 }
 
+func (s *Store) LoadNewVolumes() {
+	for _, location := range s.Locations {
+		location.loadExistingVolumes(s.NeedleMapKind)
```

Comment:
> Will this load previous volumes again?

No, it will not
https://github.com/seaweedfs/seaweedfs/blob/4f7a1f67cdecf36ae590d35eca0f8e8510fce715/weed/storage/disk_location.go#L155